bcos-boostssl/websocket/WsConnector.cpp

```diff
@@ -238,6 +244,12 @@ void WsConnector::connectToWsServer(const std::string& _host, uint16_t _port,
             auto stream = std::make_shared<boost::beast::websocket::stream<
                 boost::beast::ssl_stream<boost::beast::tcp_stream>>>(*ioc, *ctx);
 
+            // must set compress option before handshake, otherwise network compress will fail
+            boost::beast::websocket::permessage_deflate opt;
```

Comment:
you can encapsulated this as a function.

Reply:
has encapsulated a function in Wstools.h